client_test.go

```diff
@@ -515,17 +536,6 @@ func Test_Client_Stop(t *testing.T) {
 		}
 	}
 
-	t.Run("not started", func(t *testing.T) {
-		t.Parallel()
-
-		dbPool := riverinternaltest.TestDB(ctx, t)
-		client := newTestClient(t, dbPool, newTestConfig(t, nil))
-
-		err := client.Stop(ctx)
-		require.Error(t, err)
-		require.Equal(t, "client not started", err.Error())
-	})
```

Comment:
I originally left this error in, but to guarantee non-raciness, I wouldn't had to an extra mutex to protect it, and it didn't seem worth it since the error's not particularly valuable. (The error wasn't race protected before either, but now we're actually testing it so the problem was revealed.)